### ISSUE TITLE
/api/doc not present in my Kimai 2 installation

### ISSUE THREAD

**Issue by xchatter** [kevinpapst/kimai2#1798]
Hi,

I wanted to take a look at the REST API but my installation seems to not have anything under /api/doc so I get 404. I used 1-click installer present on my hosting.
Can you give me a hint how I can use this functionality?

P.S.
Sorry if I missed something in the documentation, I use Kimai for the first time. :)
I followed https://www.kimai.org/documentation/rest-api.html 

Thanks! :)

**Comment by kevinpapst** [comment]
The URL has no language inside, eg: https://demo.kimai.org/api/doc
Login credentials can be found here: https://www.kimai.org/demo/
Who is your hoster? What kind of 1-click installer?

**Comment by xchatter** [comment]
Yep, I don't add any language.
My hosting provider is SuperHosting.bg and they use Softaculous.

**Comment by xchatter** [comment]
Ah.... it is my mistake, I just noticed they provide Kimai 1.9 
I got it wrong because I noticed the update date is pretty recent so I wrongly assumed it is probably Kimai 2.

Sorry about this issue! 
I will install Kimai 2 and try again.

Thanks!

**Comment by kevinpapst** [comment]
KImai 1.9 is Kimai 2

**Comment by kevinpapst** [comment]
You can check var/log/prod.log in your Kimai directory to see if you find any clue there.
Otherwise I don't know why this URL should not work.

**Comment by xchatter** [comment]
I was able to access it on /public/api/doc .
I guess just trying /api/doc does not redirect automatically to /public/api/doc .
I will have to set some redirect rule or just use it like this.

Thanks anyway and sorry to bother you. :)

**Comment by kevinpapst** [comment]
See if you can change the "webroot/document root" of your domain to point to public/ directly, so your URL doesn not include it any more, as it is a security risk!

**Comment by xchatter** [comment]
Hi there,

Thanks for the hint. I did it and it works with just a small problem. Let's say my installation is hosted on app.com
If you go to app.com/ then you are redirected to app.com/public which now leads to application error "Page not found" because I changed where my domain is pointing to. On this page is the link to the Dashboard. The link to the dashboard leads you to app.com/en/homepage which is correct and the application starts working.

To fix this problem and to be able to properly use app.com/ I changed the following line in the .htaccess file in the root kimai folder from:
`<IfModule mod_alias.c>
RedirectMatch 301 ^/$ https://time.megdanov.com/public
</IfModule>`

to

`<IfModule mod_alias.c>
RedirectMatch 301 ^/$ https://time.megdanov.com/en/homepage
</IfModule>`

Now everything is fine.
